PWGGA/GammaConv/AliCaloPhotonCuts.cxx

```diff
@@ -1296,6 +1296,7 @@ Bool_t AliCaloPhotonCuts::ClusterIsSelectedMC(TParticle *particle,AliMCEvent *mc
 
     if ( particle->Eta() < fMinEtaCut || particle->Eta() > fMaxEtaCut ) return kFALSE;
     if ( particle->Phi() < fMinPhiCut || particle->Phi() > fMaxPhiCut ) return kFALSE;
+    if ( fClusterType == 3 && particle->Eta() < 0.227579 && particle->Eta() > -0.227579 ) return kFALSE;
 
```

Comment:
could you please make these variables setable somewhere i.e a 3rd variable for the etaCut, otherwise we will most likely forget it at some point.